quemsoueu.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+<header>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Ilse Barradas - Macrame</title>
+    <link rel="stylesheet" href="./styles.css">
+    <link rel="stylesheet" href="./reset.css">
+    <link rel="stylesheet" href="./header.css">
+    <link rel="stylesheet" href="./footer.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Poppins:wght@900&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap" rel="stylesheet">
+    <script src="main.js" defer></script>
+</header>
+
+<body>
+<div class="principal">
+
+     <!--- Primeira linha ------------------->
+     <div class="banner__superior">
+        <div class="logo__superior">
+            <img src="./img/logovinho.jpeg" class="logo__imagem">
+        </div>
+        <nav class="menu__superior">
+            <ul class="menu__superior__grupo">
+                <li><a href="paineis.html" class="menu__superior--itens">Painéis</a></li>
+                <li><a href="suporte.html" class="menu__superior--itens">Suporte de Plantas</a></li>
+                <li><a href="espelho.html" class="menu__superior--itens">Espelhos</a></li>
+                <li><a href="balanços.html" class="menu__superior--itens">Balanços</a></li>
+                <li><a href="bolsas.html" class="menu__superior--itens">Bolsas</a></li>
+                <li><a href="arvore da vida.html" class="menu__superior--itens">Árvores da vida</a></li>
+                <li><a href="quemsoueu.html" class="menu__superior--itens">Quem sou eu</a></li>
+            </ul>
+        </nav>
+
+    </div>
+
+       <!--- Quadrado Propaganda ------------------->
+       <div class="quadrado__propaganda">
+            <img src="./imgdosprodutos/testebanner1.png" class="imagem--banner" alt="Imagem 1">
+            <img src="./imgdosprodutos/testebanner2.png" class="imagem--banner" alt="Imagem 2">
+            <img src="./imgdosprodutos/testebanner3.png" class="imagem--banner" alt="Imagem 3">
+        </div>
+
+    <div class="divisoria"></div>
+
+    <div class="novidades">Quem sou eu</div>
+
+    <div class="novidades__produto">
+        <div class="novidades__produto--01">
+            <div class="novidades__produto--01--info">
+                <p class="novidades__produto--01--txt quemsoueu__titulo">Ilse Barradas</p>
+                <p class="novidades__produto--01--txt quemsoueu__txt">bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla</p>
```

Comment:
Eu sei que é só um placeholder mas to colocando um comentário pra vc não esquecer de colocar o texto final aqui =]

Reply:
Blz